skfda/exploratory/stats/_stats.py

```diff
@@ -361,3 +366,91 @@ def trim_mean(
     trimmed_curves = X[indices_descending_depth[:n_samples_to_keep]]
 
     return trimmed_curves.mean()
+
+
+def duoble_mean(X: FData) -> FData:
+    """Compute the double mean of a FData object.
```

Comment:
Move that to the next line.
```suggestion
    """
    Compute the double mean of a FData object.
```